docs/helper/docker/mxnet/base/ubuntu-18.04/Dockerfile.gpu.mx_latest

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y tzdata build-essential git ninja-build ccache libopenblas-dev \
+    libopencv-dev cmake wget python3 python3-pip python3-distutils libgomp1
+
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+# CMake 3.13 or higher is required.
+RUN pip3 install --user --upgrade "cmake==3.13.3"
+
+# Build MXNet
+WORKDIR /opt
+RUN git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet
+
+WORKDIR /opt/mxnet
+RUN cp config/linux.cmake config.cmake && \
+    rm -rf build && mkdir -p build
+
+WORKDIR /opt/mxnet/build
+RUN ~/.local/bin/cmake -GNinja .. && \
+    ~/.local/bin/cmake --build .
+
+# Install MXNet for Python
+RUN cd /opt/mxnet/python && \
+    pip3 install --user -e .
```

Comment:
We could use `pip install mxnet-cu102` to install the GPU version of MXNet, It will be much faster than building by ourselves.